### PR TITLE
add a SOCKS demo with a copy/paste-based signalling channel

### PR DESCRIPTION
**Pull request by trevj**
Also rename the parent folder to `socks-server`, in line with `echo-server`.

### CODE REVIEW

File: Gruntfile.coffee [hunk 1]
```diff
@@ -224,33 +223,31 @@ module.exports = (grunt) ->
     'copy:echoServerChromeApp'
   ]
 
-  #-------------------------------------------------------------------------
-  # tasks for sample apps
-  taskManager.add 'socksRtcNet', [
+  taskManager.add 'socksSamples', [
```
Comment (iislucas) [inline Gruntfile.coffee:231]: Maybe good to call it socksServerSamples not to confuse it with something that might be in the socks folder?

Comment (trevj) [inline Gruntfile.coffee:231]: What do you think of renaming the `socks` folder to `socks-common`, and keeping this as `socksSamples`?

**Comment by trevj** [comment]
Oops, looks like Github only supports one assignee.

File: Gruntfile.coffee (hunk 1 above)
Comment (iislucas) [inline Gruntfile.coffee:231]: that works.

Comment (trevj) [inline Gruntfile.coffee:231]: OK, I just renamed it to `socks-common`.

**Comment by matthild** [comment]
:+1:

**Pull request merged**